catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/security/ShareableMetacardSharingPolicyPlugin.java

```diff
@@ -82,12 +83,12 @@
             .collect(Collectors.toSet()));
   }
 
-  private Map<String, Set<String>> getPolicy(WorkspaceMetacardImpl workspace) {
+  private Map<String, Set<String>> getPolicy(ShareableMetacardImpl shareableMetacard) {
     return Stream.of(
-            getWorkspaceSecurityTag(),
-            getOwnerPermission(workspace),
-            getGroupPermission(workspace),
-            getIndividualPermission(workspace))
+            getSharingSecurityTag(),
```

Comment:
❓ Can we come up with a substitute for tagging since, when this stuff moves to `core`, it would result in a lot of unnecessary state?

Reply:
I see. The tag is not effectively a tag anymore, but an alias used by the policy.